packages/rum/src/domain/requestCollection.ts

```diff
@@ -109,7 +109,7 @@ export function trackFetch(lifeCycle: LifeCycle, configuration: Configuration, t
         spanId: context.spanId,
         startTime: context.startTime,
         status: context.status,
-        traceId: context.traceId,
+        traceId: context.status === 0 ? undefined : context.traceId,
```

Comment:
it could be nice to also clear spanId then.

Reply:
I agree, I will clear it.